### PR TITLE
tail: add spaces after hashes

### PR DESCRIPTION
**Pull request by bltnkl**
Add spaces after title hashes to correct formatting.

### CODE REVIEW

File: src/tail/tail.rs [hunk 1]
```diff
@@ -79,8 +79,8 @@ pub fn uumain(args: Vec<String>) -> i32 {
     opts.optflag("h", "help", "help");
     opts.optflag("V", "version", "version");
     opts.optflag("v", "verbose", "always output headers giving file names");
-    // TODO: --silent flag as alias to --quiet
     opts.optflag("q", "quiet", "never output headers giving file names");
+    opts.optflag("", "silent", "synonime of --quiet");
```
Comment (Arcterus) [inline src/tail/tail.rs]: "synonime" should be "synonym"

Comment (bltnkl) [reply src/tail/tail.rs]: Fixed.

**Comment by bltnkl** [comment]
I squashed two commits, fixed the typo too.

**Comment by bltnkl** [comment]
It's now one commit.

**Comment by Arcterus** [comment]
Can you adjust the commit message to have "tail: " in the front?  So like "tail: squashed the following commits:" or something.

**Comment by bltnkl** [comment]
Fixed title.

**Pull request merged**